src/SprykerEco/Shared/Adyen/AdyenConstants.php

```diff
@@ -36,6 +36,14 @@ interface AdyenConstants
     public const IDEAL_ISSUERS_LIST = 'ADYEN:IDEAL_ISSUERS_LIST';
     public const REQUEST_CHANNEL = 'ADYEN:REQUEST_CHANNEL';
 
+    /**
+     * Specification:
+     * - Return URL for Klarna payment method.
+     *
+     * @api
+     */
+    public const KLARNA_RETURN_URL = 'ADYEN:KLARNA_RETURN_URL';
```

Comment:
Are we sure that this value env specific?

Reply:
Yes like other return redirects.